src/cljs/org/broadinstitute/firecloud_ui/paths.cljs

```diff
@@ -45,5 +45,8 @@
 (defn submission-details [workspace-id submission-id]
   (str (submissions-list workspace-id) "/" submission-id))
 
+(defn get-abort-submission-path [workspace-id submission-id]
+  (submission-details workspace-id submission-id))
+
```

Comment:
Idiomatic REST does call for the same path for both GET/DELETE and similarly for PUT/POST/PATCH. The difference is just the method.